### PR TITLE
update designer docs content

### PR DESCRIPTION
**Pull request by konopkja**
changing list of links to a table

### CODE REVIEW

File: src/content/developers/docs/design-and-ux/index.md [hunk 1]
```diff
@@ -21,20 +21,22 @@ Effective design goes beyond creating visually appealing user interfaces. It inv
...
-- [Usability study: Multi App Staking](<https://github.com/threshold-network/UX-User-Research/blob/main/Multi-App%20Staking%20(MAS)/iterative-user-study/MAS%20Iterative%20User%20Study.pdf>)
-- [2022 DAO Research Update: What do DAO Builders Need?](https://blog.aragon.org/2022-dao-research-update/)
-- [ConSensys: DeFi User Research Report 2022](https://cdn2.hubspot.net/hubfs/4795067/ConsenSys%20Codefi-Defi%20User%20ResearchReport.pdf)
-- [Bitcoin UX report](https://github.com/patestevao/BitcoinUX-report/blob/master/report.md)
-- [Metaverse: User Research Report](https://www.politico.com/f/?id=00000187-7685-d820-a7e7-7e85d1420000)
-- [Going on Safari: Researching Users in the Metaverse](https://archive.devcon.org/archive/watch/6/going-on-safari-researching-users-in-the-metaverse/?tab=YouTube) (video, 27 min)
+| Area of focus               | Name                                              |
+| :------------------ | :------------------------------------------------------- |
+|               | [CRADL: Crypto Research and Design Lab](https://project-cradl.notion.site/Crypto-Research-and-Design-Lab-50a7127f34ed4c88ad95c7cedf7fbe36)                     |
```
Comment (minimalsm) [inline src/content/developers/docs/design-and-ux/index.md:26]: First item is blank

Screenshot 2023-09-05 at 09 38 36

File: src/content/developers/docs/design-and-ux/index.md [hunk 2]
```diff
@@ -21,20 +21,22 @@ Effective design goes beyond creating visually appealing user interfaces. It inv
...
+| `Crypto onboarding`             | [CRADL: UX in Cryptocurrency](https://docs.google.com/presentation/d/1s2OPSH5sMJzxRYaJSSRTe8W2iIoZx0PseIV-WeZWD1s/edit?usp=sharing)                        |
```
Comment (minimalsm) [inline src/content/developers/docs/design-and-ux/index.md:27]: Why are we using code backticks in the table?

```suggestion
| Crypto onboarding             | [CRADL: UX in Cryptocurrency](https://docs.google.com/presentation/d/1s2OPSH5sMJzxRYaJSSRTe8W2iIoZx0PseIV-WeZWD1s/edit?usp=sharing)                        |
```

File: src/content/developers/docs/design-and-ux/index.md [hunk 3]
```diff
@@ -21,20 +21,22 @@ Effective design goes beyond creating visually appealing user interfaces. It inv
...
-- [Usability and user satisfaction survey - Ethereum.org](https://lookerstudio.google.com/reporting/0a189a7c-a890-40db-a5c6-009db52c81c9)
-- [Usability study: Coverage pools](https://github.com/threshold-network/UX-User-Research/tree/main/Keep%20Coverage%20Pool)
-- [Usability study: Multi App Staking](<https://github.com/threshold-network/UX-User-Research/blob/main/Multi-App%20Staking%20(MAS)/iterative-user-study/MAS%20Iterative%20User%20Study.pdf>)
-- [2022 DAO Research Update: What do DAO Builders Need?](https://blog.aragon.org/2022-dao-research-update/)
-- [ConSensys: DeFi User Research Report 2022](https://cdn2.hubspot.net/hubfs/4795067/ConsenSys%20Codefi-Defi%20User%20ResearchReport.pdf)
-- [Bitcoin UX report](https://github.com/patestevao/BitcoinUX-report/blob/master/report.md)
-- [Metaverse: User Research Report](https://www.politico.com/f/?id=00000187-7685-d820-a7e7-7e85d1420000)
-- [Going on Safari: Researching Users in the Metaverse](https://archive.devcon.org/archive/watch/6/going-on-safari-researching-users-in-the-metaverse/?tab=YouTube) (video, 27 min)
+| Area of focus               | Name                                              |
```
Comment (minimalsm) [inline src/content/developers/docs/design-and-ux/index.md:24]: Separate from this PR, the table implementation is causing horizontal scroll on mobile.

Screenshot 2023-09-05 at 09 42 23

cc: @pettinarip @TylerAPfledderer

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks @konopkja, a few issues that need fixing below.

In the bigger picture sense, what is the reasoning/motivation for making this a table? 

It seems like an arbitrary decision when to use a table vs. when not.

File: src/content/developers/docs/design-and-ux/index.md (hunk 1 above)
Comment (konopkja) [reply src/content/developers/docs/design-and-ux/index.md:26]: its a repository not a single study

File: src/content/developers/docs/design-and-ux/index.md (hunk 3 above)
Comment (TylerAPfledderer) [reply src/content/developers/docs/design-and-ux/index.md:24]: @pettinarip not showing in Firefox, but definitely in Chrome. Looks to be due to the `min-width` set on the table.

File: src/content/developers/docs/design-and-ux/index.md (hunk 2 above)
Comment (konopkja) [reply src/content/developers/docs/design-and-ux/index.md:27]: hmm maybe i made a mistake, was trying to differentiate between a study and a category which i did like that, can we do it any other way?

**Comment by konopkja** [comment]
> > Thanks @konopkja, a few issues that need fixing below.
> > In the bigger picture sense, what is the reasoning/motivation for making this a table?
> > It seems like an arbitrary decision when to use a table vs. when not.
> 
> For such a list I thought it will be easier to follow than classic list.
> 
> divided the research by use case as i assume people will be interested mostly only to whats relevant to them

in future its possible i will divide more of the content into tables, i think its more user friendly. Any final pushback or comments? @minimalsm

File: src/content/developers/docs/design-and-ux/index.md (hunk 2 above)
Comment (konopkja) [reply src/content/developers/docs/design-and-ux/index.md]: changed this to tags

**Comment by konopkja** [comment]
@corwintines  tags worked, only color doesnt but i think thats ok for now! Would like to have this merger if @minimalsm  doesnt have any strong pushback

**Review by corwintines — APPROVED** [review]
@konopkja I added some variant support for Badge in our component system to do the colors.

@TylerAPfledderer @pettinarip wondering what you think about this if you don't mind.

**Comment by pettinarip** [comment]
@corwintines @corwintines given that the Badge component is not part of the DS I'd be ok with this implementation. However, I'm wondering if we could use [the new `Tag` component](https://dev--63b7ea99632763723c7f4d6b.chromatic.com/?path=/story/molecules-display-content-tags--style-variants-basic) here. What do you think?

Having said that, defining a variant per color is incorrect. [Variants](https://chakra-ui.com/docs/styled-system/component-style) purpose is to define visual styles. Ideally, once you select the variant you want to use (e.g. solid or outline), you can pick what color scheme to use on top of that. For example, `<Badge variant='outline' colorScheme='green'>`

**Review by pettinarip — APPROVED** [review]

**Comment by pettinarip** [comment]
@corwintines I've implemented the approach I described before. Merging this now.

**Pull request merged**